config.yml

```diff
@@ -2,8 +2,8 @@ version: 1
 accounts:
   - name: treasury
     coins:
-      - 100000000uatom
-      - 100000000uusdt
+      - 1000000000000uatom
+      - 1000000000000uusdc
```

Comment:
Can we keep both uusdc and uusdt? (uusdt was used on price-feeder ATOM/USDT pool testing locally)

Reply:
done